polylogue/cli/search_cli.py

```
@@ -283,13 +281,11 @@ def _open_single_search_hit(ui, hit: SearchHit) -> None:
 
     target_obj = Path(target_path)
     is_html = target_obj.suffix.lower() == ".html"
-    if is_html and anchor_label:
-        try:
-            webbrowser.open(target_obj.as_uri() + f"#{anchor_label}")
-            ui.console.print(f"[dim]Opened {target_obj}#{anchor_label} in browser[/dim]")
+    if is_html:
+        if open_in_browser(target_obj, anchor_label):
```

Ruff (0.15.9)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)